src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -17,13 +17,29 @@ public class CommandResult {
     /** The application should exit. */
     private final boolean exit;
 
+    /** Reminder information should be shown to the user. */
+    private final boolean showReminder;
```

Comment:
Perhaps a more intuitive boolean name can be used here

Reply:
The intent of the variable is to mark whether to show reminder window or not similar to help window. Hence I think that it signals the intent sufficiently